tests/baselines/reference/docker/vscode.log

```diff
@@ -4,23 +4,25 @@ yarn run vX.X.X
 $ gulp compile --max_old_space_size=4095
 [XX:XX:XX] Node flags detected: --max_old_space_size=4095
 [XX:XX:XX] Using gulpfile /vscode/gulpfile.js
-[XX:XX:XX] Error: /vscode/extensions/typescript-language-features/src/protocol.d.ts(13,3): Duplicate identifier 'PrepareCallHierarchy'.
-[XX:XX:XX] Error: /vscode/extensions/typescript-language-features/src/protocol.d.ts(14,3): Duplicate identifier 'ProvideCallHierarchyIncomingCalls'.
-[XX:XX:XX] Error: /vscode/extensions/typescript-language-features/src/protocol.d.ts(15,3): Duplicate identifier 'ProvideCallHierarchyOutgoingCalls'.
+[XX:XX:XX] Error: /vscode/src/vs/workbench/contrib/remote/browser/remote.ts(637,8): Parameter 'choice' implicitly has an 'any' type.
```

Comment:
these are new

Reply:
this is a correct error; the target signature has no parameters so `choice` doesn't have anywhere to get a type from.

The mystery is why we didn't report this earlier.

Reply:
`choice` is marked optional on the argument (`(choice?) => ...`), so I guess we're reporting this error on optionals now. This seems correct and not too bad a breaking change. However, @DanielRosenwasser should know about it in case we need to report it in the notes.